code/game/machinery/computer/camera.dm

```diff
@@ -13,21 +13,30 @@ var/global/list/tv_monitors = list()
 	var/mapping = 0//For the overview file, interesting bit of code.
 
 	var/list/datum/action/camera/our_actions = list()
+	var/static/list/obj/machinery/camera/sorted_cams = list()
+	var/static/list/obj/machinery/camera/deactivated_cams = list()
 
 	light_color = LIGHT_COLOR_RED
 
 /obj/machinery/computer/security/New()
 	..()
-	var/datum/action/camera/previous/P = new(src, current)
-	var/datum/action/camera/cancel/C = new(src, current)
-	var/datum/action/camera/listing/L = new(src, current)
-	var/datum/action/camera/next/N = new(src, current)
+	var/datum/action/camera/previous/P = new(src)
+	var/datum/action/camera/cancel/C = new(src)
+	var/datum/action/camera/listing/L = new(src)
+	var/datum/action/camera/next/N = new(src)
 	our_actions = list(P, C, L, N)
-	var/list/camera_list = get_cameras()
-	if (camera_list.len)
-		current = camera_list[1]
+
+	if (ticker && ticker.mode == GAME_STATE_PLAYING)
+		initialize()
+
 	tv_monitors += src
 
+/obj/machinery/computer/security/initialize()
```

Comment:
For some reason this doesn't get called, or not properly.
The sorted_cams list is blank on roundstart

I suspect that it's called before the cameranet himself is initalised